src/de/jost_net/JVerein/gui/view/MailVorlageDetailView.java

```diff
@@ -54,4 +66,49 @@ public void handleAction(Object context)
     }, null, true, "document-save.png");
     buttons.paint(this.getParent());
   }
+
+  @Override
+  public void handleEvent(Event event)
+  {
+     if (event.keyCode == org.eclipse.swt.SWT.F6)
+    {
+      new OpenInsertVariableDialogAction().handleAction(null);
+    }
+  }
+
+  private static final class OpenInsertVariableDialogAction implements Action
+  {
+
+    @Override
+    public void handleAction(Object context)
+    {
+      try
+      {
+        LesefeldAuswerter lesefeldAuswerter = new LesefeldAuswerter();
```

Comment:
Die Lesefelder werden in der MitgliedMap automaitsch mit ausgewertet, da muss hier nicht extra gemacht werden

Reply:
Werde ich entfernen

Reply:
Werde ich entfernen

Reply:
Habe ich in der aktualisierten Form drinnen gelassen, da die OpenInsertVariableDialogAction nach dem Refactor einen  Context vom Typ LesefeldAuswerter braucht.